.buildkite/pipeline.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - label: hydra-eval-errors
+    command: 'nix-build ./nix -A iohkNix.hydraEvalErrors && ./result/bin/hydra-eval-errors.py'
+    agents:
+      system: x86_64-linux
+
+  - label: 'check-cabal-project'
+    command: 'nix-build ./nix -A iohkNix.checkCabalProject -o check-cabal-project.sh && ./check-cabal-project.sh'
+    agents:
+      system: x86_64-linux
+
+  - label: 'stack-cabal-sync'
+    command: 'nix-shell ./nix -A iohkNix.stack-cabal-sync-shell --run .buildkite/stack-cabal-sync.sh'
+    agents:
+      system: x86_64-linux
```

Comment:
Please also add the build and push job, as per the reference: https://github.com/input-output-hk/cardano-node/blob/master/.buildkite/pipeline.yml#L41-L48